web/frontend/src/components/slam.vue

```diff
@@ -169,17 +191,30 @@ const executeMoveOnMap = async () => {
     new grpc.Metadata(),
     (error: ServiceError | null, response: motionApi.MoveOnMapResponse | null) => {
       if (error) {
-        moveClick = !moveClick;
+        allowMove = !allowMove;
+        executeDeleteDestinationMarker();
         toast.error(`Error moving: ${error}`);
         return;
       }
-      moveClick = !moveClick;
+      allowMove = !allowMove;
```

Comment:
I don't think `allowMove` should ever be set. It is inherently determined based on its dependencies via $computed((). Flipping the `bool` here will either break its contract with its dependencies. In addition once `updateDest` gets set to false in `executeDeleteDestinationMarker`, it will override the new definition of allowMove making it useless to set it here.